transaction-pool/src/tests/helpers.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+use std::cmp;
+use std::collections::HashMap;
+
+use {scoring, Scoring, Ready, Readiness, Address as Sender};
+use super::{Transaction, SharedTransaction, U256};
+
+#[derive(Default)]
+pub struct DummyScoring;
+
+impl Scoring<Transaction> for DummyScoring {
+	type Score = U256;
+
+	fn compare(&self, old: &Transaction, other: &Transaction) -> cmp::Ordering {
+		old.nonce.cmp(&other.nonce)
+	}
+
+	fn choose(&self, old: &Transaction, new: &Transaction) -> scoring::Choice {
+		let decision = if old.nonce == new.nonce {
+			if new.gas_price > old.gas_price {
+				scoring::Choice::ReplaceOld
+			} else {
+				scoring::Choice::RejectNew
+			}
+		} else {
+			scoring::Choice::InsertNew
+		};
+
+		decision
```

Comment:
Since this method is small and intentions are crystal clear, I may suggest omitting the `decision` value. Two nested `if`s would be OK, in my opinion.